packages/components/src/List/Toolbar/ColumnChooser/Content/DefaultColumnChooser.components.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import theme from './ColumnChooser.scss';
+import ActionButton from '../../../../Actions/ActionButton';
+import ColumnDisplayer from '../ColumnDisplayer';
+
+export const DefaultHeader = ({ t }) => (
+	<div className={classNames(theme['tc-column-chooser-header'], 'tc-column-chooser-header')}>
+		{t('COLUMN_CHOOSER_HEADER_TITLE', {
+			defaultValue: 'Modify columns position',
+		})}
+	</div>
+);
+
+DefaultHeader.propTypes = {
+	t: PropTypes.func.isRequired,
+};
+
+export const getColumnDisplay = (length, onChangeVisibility, onBlurOrder, onKeyPressOrder, t) => (
+	column,
+	index,
+) => {
+	const displayerProps = {
+		...column,
+		length,
+		onBlurOrder: onBlurOrder(index),
+		onChangeVisibility: onChangeVisibility(index),
+		onKeyPressOrder: onKeyPressOrder(index),
+		t,
+	};
+	return <ColumnDisplayer key={column.label} {...displayerProps} />;
+};
+
+export const DefaultBody = ({ columns, onChangeVisibility, onBlurOrder, onKeyPressOrder, t }) => (
+	<div
+		id="column-chooser-content"
```

Comment:
Please generate an id from a props.id.
This implies that it can't be used twice in a page.